### DISCUSSION TITLE
HPool and Chia Miner PP. Offcial Protocol?

### DISCUSSION THREAD

**Discussion by MaximusChia** [Chia-Network/chia-blockchain#7689] [General]
Looking through miningpoolstats, there is a Hpool, now named iHpool, which claims to have Official protocol working...
However, after revieweing its page, the only instruction is to download a program called ChiaMiner PP 
https://github.com/hpool-dev/chiapp-miner/releases
What the hell is that? How an Official Chia protocol requieres you to use a external program?
Why there isnt an url  to join via NFT as other pools do?

**Comment by keliew** [comment]
Their url is most likely hardcoded in the software...

Instead of having it as a web-based application, they opted for a desktop-based solution.
